local_repo/roles/parse_and_download/files/parse_and_download.py

```diff
@@ -222,6 +222,7 @@ def main():
     status_file_path = os.environ.get('STATUS_FILE_PATH')
     nerdctl_registry_host = os.environ.get('NERDCTL_REGISTRY_HOST')
     software_name = os.environ.get('SOFTWARE_NAME')
+    openssl_cert_path = os.environ.get('OPENSSL_CERT_PATH')
```

Comment:
what if the environment variable is not set? Shall it have a default value?

Reply:
we are setting this environment variable before calling this python file